moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,169 +1,200 @@
-find_package(catkin REQUIRED COMPONENTS
-  moveit_core
-  moveit_ros_planning
-  moveit_ros_visualization
-  rviz
-  rosconsole
-  roscpp
-  urdf
-  srdfdom
-)
 find_package(ompl REQUIRED)
-include_directories(SYSTEM ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${OMPL_INCLUDE_DIRS})
-include_directories(include)
 
 # Qt Stuff
-if(rviz_QT_VERSION VERSION_LESS "5")
-  find_package(Qt4 ${rviz_QT_VERSION} REQUIRED QtCore QtGui)
-  include(${QT_USE_FILE})
-else()
-  find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
-  set(QT_LIBRARIES Qt5::Widgets)
-endif()
+find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
+set(QT_LIBRARIES Qt5::Widgets)
+
 # Instruct CMake to run moc automatically when needed.
 set(CMAKE_AUTOMOC ON)
 add_definitions(-DQT_NO_KEYWORDS)
 
-# Support new yaml-cpp API.
-find_package(PkgConfig)
-pkg_check_modules(YAMLCPP REQUIRED yaml-cpp>=0.5)
-include_directories(SYSTEM ${YAMLCPP_INCLUDE_DIRS})
-
-catkin_package(
-  INCLUDE_DIRS
-    include
-  LIBRARIES
-    ${PROJECT_NAME}_tools
-  CATKIN_DEPENDS
-    moveit_ros_planning
-    moveit_ros_visualization
-    roscpp
-)
-
 # Header files that need Qt Moc pre-processing for use with Qt signals, etc:
 set(HEADERS
   src/widgets/navigation_widget.h
   src/widgets/header_widget.h
   src/widgets/setup_assistant_widget.h
   src/widgets/start_screen_widget.h
-  src/widgets/planning_groups_widget.h
-  src/widgets/double_list_widget.h
-  src/widgets/kinematic_chain_widget.h
-  src/widgets/group_edit_widget.h
-  src/widgets/default_collisions_widget.h
-  src/widgets/robot_poses_widget.h
-  src/widgets/end_effectors_widget.h
-  src/widgets/virtual_joints_widget.h
-  src/widgets/passive_joints_widget.h
-  src/widgets/perception_widget.h
-  src/widgets/ros_controllers_widget.h
-  src/widgets/controller_edit_widget.h
-  src/widgets/configuration_files_widget.h
+#  src/widgets/planning_groups_widget.h
+#  src/widgets/double_list_widget.h
+#  src/widgets/kinematic_chain_widget.h
+#  src/widgets/group_edit_widget.h
+#  src/widgets/default_collisions_widget.h
+#  src/widgets/robot_poses_widget.h
+#  src/widgets/end_effectors_widget.h
+#  src/widgets/virtual_joints_widget.h
+#  src/widgets/passive_joints_widget.h
+#  src/widgets/perception_widget.h
+#  src/widgets/ros_controllers_widget.h
+#  src/widgets/controller_edit_widget.h
+#  src/widgets/configuration_files_widget.h
   src/widgets/setup_screen_widget.h
-  src/widgets/simulation_widget.h
-  src/widgets/author_information_widget.h
+#  src/widgets/simulation_widget.h
+#  src/widgets/author_information_widget.h
 )
 
 # Tools Library
 add_library(${PROJECT_NAME}_tools
   src/tools/compute_default_collisions.cpp
   src/tools/moveit_config_data.cpp
-  src/tools/collision_linear_model.cpp
-  src/tools/collision_matrix_model.cpp
-  src/tools/rotated_header_view.cpp
+#  src/tools/collision_linear_model.cpp
+#  src/tools/collision_matrix_model.cpp
+#  src/tools/rotated_header_view.cpp
 )
```

Comment:
I think @MarqRazz already told you how to put this together.  Please move this into a separate package instead of into one with a bunch of commented-out code.  That way it will be much easier to review.